### PR TITLE
Feature/point lights

### PR DESCRIPTION
**Pull request by David-Hanna**

### CODE REVIEW

File: Assets/Kart/kart.fsh [hunk 1]
```diff
@@ -32,7 +37,30 @@ out vec4 FragmentColor;
...
+
+    //-----------------------------------------
+    // Point light computation.
+
+    vec3 PointLightDirectionVector = PointLightPosition - v_position.xyz;
+    float distance = length(PointLightDirectionVector);
+
+    vec3 PointLight = vec3(0.0, 0.0, 0.0);
+
+    //if (distance <= PointLightRange)
```
Comment (MatthewWhite) [inline Assets/Kart/kart.fsh:54]: Is there a reason this code is commented out but still in the file?

**Review by MatthewWhite — APPROVED** [review]

**Review by MatthewWhite — APPROVED** [review]
Just resolve conflicts, then good to go

**Review by MatthewWhite — APPROVED** [review]
Why does it let me approve multiple times?

File: Assets/Kart/kart.fsh (hunk 1 above)
Comment (David-Hanna) [reply Assets/Kart/kart.fsh:54]: It's there just to be able to un-comment it if we want a limit on the range of the point light. The PointLightRange parameter is still being passed to use if necessary. Having a range is how we were taught to use point lights but I've never really seen a use for it considering it falls to zero after a certain point in the attenuation calculation anyway and it adds a branch to the fragment shader code.

**Pull request merged**